data/presets/railway/_halt.json

```diff
@@ -7,6 +7,6 @@
     "tags": {
         "railway": "halt"
     },
-    "name": "Train Station (Halt / Request)",
+    "name": "Train Station (Halt/Request)",
```

Comment:
How about simplifying this to “Limited-Service Train Station”?

I don’t think any dialect of American English calls these “halts” or knows what a halt is. On the other hand, the U.S. community isn’t always using `railway=halt` to indicate a request stop (flag stop); that’s what [`request_stop=yes`](https://wiki.openstreetmap.org/wiki/Tag:request_stop%3Dyes) is for. Many occurrences of `railway=halt` in the U.S. indicate a limited-service station stop, for example, only on weekends or during football games, while some entire light rail systems operate exclusively on a request-stop model, including at their major stations.

I think “limited-service” would capture the intent of both the German and North American usages of this tag. We should add a field for `request_stop` while we’re at it: #875.